src/components/[guild]/RoleCard/RoleCard.tsx

```diff
@@ -54,92 +83,165 @@ const RoleCard = memo(({ role }: Props) => {
           }
         : {})}
     >
-      <SimpleGrid columns={{ base: 1, md: 2 }}>
-        <Flex
-          direction="column"
-          p={5}
-          borderRightWidth={{ base: 0, md: 1 }}
-          borderRightColor={colorMode === "light" ? "gray.200" : "gray.600"}
-        >
-          <HStack justifyContent="space-between" mb={6} spacing={3}>
-            <HStack spacing={4} minW={0}>
-              <GuildLogo
-                imageUrl={role.imageUrl}
-                size={{ base: "48px", md: "52px" }}
-              />
-              <Heading
-                as="h3"
-                fontSize="xl"
-                fontFamily="display"
-                minW={0}
-                overflowWrap={"break-word"}
+      <Collapse in={isOpen} startingHeight={collapsedHeight}>
+        <SimpleGrid columns={{ base: 1, md: 2 }}>
+          <Flex
+            direction="column"
+            p={5}
+            borderRightWidth={{ base: 0, md: 1 }}
+            borderRightColor={
+              !isOpen
+                ? "transparent"
```

Comment:
I think we could use the `useColorModeValue` hook in this file, so we could remove the double ternary here (it's a bit hard to parse, at least for me, haha)

So I'd do something like this:
```ts
const requirementsListBgColor = useColorModeValue("gray.50", "blackAlpha.300")
const roleDetailsBorderColor = useColorModeValue("gray.200", "gray.600")
  ```

Reply:
True, I agree, switched to invidual hooks